File: services/src/Moe.La.Core/Enums/MoamalaStatuses.cs
using Moe.La.Common.Extensions;
using Moe.La.Common.Resources.Common;

namespace Moe.La.Core.Enums
{
    public enum MoamalaStatuses
    {
        /// <summary>
        /// جديدة
        /// </summary>
        [LocalizedDescription("New", typeof(EnumsLocalization))]
        New = 1,

        /// <summary>
        /// محالة
        /// </summary>
        [LocalizedDescription("Referred", typeof(EnumsLocalization))]
        Referred = 2,

        /// <summary>
        /// مسندة
        /// </summary>
        [LocalizedDescription("Assigned", typeof(EnumsLocalization))]
        Assigned = 3,

        /// <summary>
        /// معادة
        /// </summary>
        [LocalizedDescription("MoamalaReturned", typeof(EnumsLocalization))]
        MoamalaReturned = 4,
    }
}

File: website/Moe.La.Web/els-app/src/app/core/enums/MoamalaStatuses.ts
export enum MoamalaStatuses {
  /// <summary>
  /// جديدة
  /// </summary>
  New = 1,

  /// <summary>
  /// محالة
  /// </summary>
  Referred = 2,

  /// <summary>
  /// مسندة
  /// </summary>
  Assigned = 3,

  /// <summary>
  /// معادة
  /// </summary>
  MoamalaReturned = 4,
}

export enum MoamalaSteps
{

    /// <summary>
    /// جديدة أو معادة إلى موزع المعاملات و المشرف
    /// </summary>
    Initial = 1,

    /// <summary>
    /// إحالة /معادة إلى إدارة عامة/منطقة
    /// </summary>
    Branch = 2,

    /// <summary>
    /// إحالة/ معادة إلى إدارة مختصة
    /// </summary>
    Department = 3,

    /// <summary>
    /// مسندة إلى موظف
    /// </summary>
    Employee = 4


}
